src/Microsoft.DotNet.Interactive.Formatting/DefaultHtmlFormatterSet.cs

```diff
@@ -74,38 +44,70 @@ protected override bool TryInferFormatter(Type type, out ITypeFormatter formatte
                     view.WriteTo(writer, HtmlEncoder.Default);
                 }),
 
-                [typeof(HtmlString)] = new HtmlFormatter<HtmlString>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
+                new HtmlFormatter<HtmlString>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
 
-                [typeof(JsonString)] = new HtmlFormatter<JsonString>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
+                new HtmlFormatter<JsonString>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
 
-                [typeof(PocketView)] = new HtmlFormatter<PocketView>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
+                new HtmlFormatter<PocketView>((view, writer) => view.WriteTo(writer, HtmlEncoder.Default)),
```

Comment:
Even prior to these changes, I believe all three of these cases should be replaceable by a single formatter for `IHtmlString`.

Reply:
that would be great